docs/validator/validator-faq.md

```diff
@@ -48,22 +71,14 @@ Instead NEAR allows smart contracts to stake, because in NEAR contracts and acco
 
 Thus, if validators want to accept delegated stake, they must deploy a contract with specific rules of how delegation and reward splitting works and advertise that contract as destination to delegate.
 
-## What is a minimum amount to stake as a validator?
-
-Near, Inc. has staked 20 NEAR, so staking 10 NEAR should be enough on TestNet.
-
-## What are the slashing conditions?
-
-The only slashing conditions is signing two blocks at the same height.
-
 ## Where can I find the neardev/ folder?
 
 Once you run 'near login', a folder, called 'neardev', will be created in the directory in which you ran 'near login'.
 
-## Why did my node get kicked-out of the validation process?
+## Why did my node got kicked-out of the validation process on TestNet?
 
-You might be kicked out because your node is not producing enough blocks. Please try again and reach out to us if you are experiencing reoccurring issues.
+You might be kicked out because your node is not producing enough blocks. Please try again or open an issue on GitHub if you are experiencing reoccurring issues. Please note that sometimes we had to reset the TestNet, and nodes might need to be reinstalled to work properly.
```

Comment:
If we were to reset testnet we will announce this a couple days beforehand to make sure that all validators have enough time to prepare for it.